sparkly/utils.py

```diff
@@ -93,6 +181,45 @@ def kafka_get_topics_offsets(host, topic, port=9092):
     return offsets
 
 
+class lru_cache(object):
```

Comment:
can anyone please provide some concrete real case, just don't get how this may be useful